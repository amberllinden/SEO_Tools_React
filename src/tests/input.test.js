import React from 'react';
import renderer from 'react-test-renderer';
import Input from '../components/input';

test('Input Component - with wrapper', () => {
    const tree = renderer
        .create(
            <Input
                onChangeFunc={() => 'true'}
                isWrapped
                inputType='text'
                value='value'
            />)
        .toJSON();
    expect(tree).toMatchSnapshot();
});

test('Input Component â€“ without wrapper, with placeholder', () => {
    const tree = renderer
        .create(
            <Input
                onChangeFunc={() => 'false'}
                isWrapped={false}
                value='1234wdwd'
                placeholder="Placeholder Text"
            />)
        .toJSON();
    expect(tree).toMatchSnapshot();
});