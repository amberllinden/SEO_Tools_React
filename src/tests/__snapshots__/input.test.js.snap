// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input Component - with wrapper 1`] = `
<div>
  <input
    onChange={[Function]}
    type="text"
    value="value"
  />
</div>
`;

exports[`Input Component â€“ without wrapper, with placeholder 1`] = `
<input
  onChange={[Function]}
  placeholder="Placeholder Text"
  value="1234wdwd"
/>
`;
